### ISSUE TITLE
VS2017 GitVersionTask for netstandard13 yields duplicate attributes on Build, foiled by AssemblyInfo autogen

### ISSUE THREAD

**Issue by tofutim** [GitTools/GitVersion#1231]
I'm quite fond of GitVersionTask. Sadly, I'm seeing some issues with VS2017. To replicate the issue,

1. Create a new .NET Standard project, switch to netstandard13 (default is netstandard14)
2. In the csproj, add
`    <PackageTargetFallback Condition=" '$(TargetFramework)' == 'netstandard1.3' ">portable-net45+win81</PackageTargetFallback>
`
3. Add GitVersionTask from NuGet - here version 3.6.5 (365!)
4. git init
5. Build

![image](https://user-images.githubusercontent.com/157766/26912047-8ba8808c-4bc6-11e7-911d-42e686ec95f8.png)

Note: This worked in VS2015. Though I had to comment out the properties. Doesn't seem that there is a way to do so in the VS2015 library (no Properties folder). Any tips much appreciated.

**Update.** Looks like it is possible to prevent autogeneration of the AssemblyInfo per https://stackoverflow.com/questions/42138418/equivalent-to-assemblyinfo-in-dotnet-core-csproj though I wonder what this means for the nuget pack feature. Looks like autogeneration comes from the nuget package specifications in Properties.

csproj looks like

```
<Project Sdk="Microsoft.NET.Sdk">

  <PropertyGroup>
    <TargetFramework>netstandard1.3</TargetFramework>
    <PackageTargetFallback Condition=" '$(TargetFramework)' == 'netstandard1.3' ">portable-net45+win81</PackageTargetFallback>
  </PropertyGroup>

  <ItemGroup>
    <PackageReference Include="GitVersionTask" Version="3.6.5" />
  </ItemGroup>

</Project>

```

I wonder if the only solution is to give on GitVersionTask and run it GitVersion after the fact but before compiling.... although I guess if it is not auto-generated Gitversion won't be able to modify it???

**Update2.** Looks like the only solution for now is

`    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
`

**Comment by SimonCropp** [comment]
Try disabling the new assembly info generation feature https://github.com/Particular/NServiceBus.Ninject/blob/develop/src/NServiceBus.Ninject/NServiceBus.Ninject.csproj#L7

**Comment by bording** [comment]
If you don't want to disable the assemblyinfo generation feature entirely, you can instead add the following to your project file:
```
<GenerateAssemblyFileVersionAttribute>false</GenerateAssemblyFileVersionAttribute>
<GenerateAssemblyInformationalVersionAttribute>false</GenerateAssemblyInformationalVersionAttribute>
<GenerateAssemblyVersionAttribute>false</GenerateAssemblyVersionAttribute>
```

**Comment by bording** [comment]
@tofutim In addition to the workarounds listed here, this is fixed in the latest 4.0 beta release, though there are some side effects: #1241, #1242.

**Comment by tofutim** [comment]
mil gracias!

**Comment by gregpakes** [comment]
Is this actually resolved?  I am using the Azure DevOps task (4.0.0.4) and it doesn't work out of the box with the new project system.  I have to put these into every project to avoid the Duplicate Attribute exception:

```
<GenerateAssemblyFileVersionAttribute>false</GenerateAssemblyFileVersionAttribute>
<GenerateAssemblyInformationalVersionAttribute>false</GenerateAssemblyInformationalVersionAttribute>
<GenerateAssemblyVersionAttribute>false</GenerateAssemblyVersionAttribute>
```

Is that expected?
